homeassistant/helpers/entity.py

```diff
@@ -311,7 +311,9 @@ def _async_write_ha_state(self):
 
         start = timer()
 
-        attr = self.capability_attributes or {}
+        attr = self.capability_attributes
+        attr = dict(attr) if attr else {}
```

Comment:
not sure if `attr = attr.copy() if attr else {}` is more readable. Otherwise, you need to look twice to see it make a copy. However, that's not an issue more a style question

Reply:
or `attr = dict(self.capability_attributes or {})`